ruby/lib/helix_runtime/build_task.rb

```diff
@@ -66,27 +66,28 @@ def define
         sh env, "cargo rustc --release #{extra_args}"
       end
 
-      task "#{name}:cargo:clean" do
+      task "cargo:clean" do
         sh "cargo clean"
       end
 
       directory lib_path
 
-      task "clobber" => "#{name}:cargo:clean"
+      task :clobber" => "cargo:clean"
 
       libfile_prefix = IS_WINDOWS ? '' : 'lib'
       native_path = "#{lib_path}/native.#{Platform.dlext}"
       native_lib = "#{libfile_prefix}#{name}.#{Platform.libext}"
 
-      file native_path => [lib_path, "#{name}:cargo:build"] do
+      file native_path => [lib_path, "cargo:build"] do
         cp "#{build_path}/#{native_lib}", native_path
       end
       CLOBBER.include(native_path)
 
-      desc "Build Helix Library for #{name}"
-      task "#{name}:build" => native_path
+      desc "Build #{name}"
+      task :build => native_path
```

Comment:
I kind of like the idea of maintaining a namespace here but add a simple alias. That way we avoid naming conflicts with the user's own Rake tasks.